docs/source/common_issues.rst

```diff
@@ -472,3 +472,41 @@ Here's the above example modified to use ``MYPY``:
 
    def listify(arg: 'bar.BarClass') -> 'List[bar.BarClass]':
        return [arg]
+
+Properties & Incompatible Types in Assignment
+---------------------------------------------
+
+Consider the code below:
```

Comment:
I think the whole thing is a bit too verbose. You should start with something like "The type of a property is taken from the getter method." Then the example should be more compact (compare some of the other sections in this file). The method bodies could just be `...` and the "main" section shouldn't have `if __name__...` .